### ISSUE TITLE
Getting java related error on ubuntu 20.04

### ISSUE THREAD

**Issue by D3SL** [miraisolutions/xlconnect#108]
While dockerizing a shiny app I made I've run into a problem involving XLConnect. My app loads in a local empty but preformatted workbook as a template, writes in data, and then outputs as a download a formatted report. On Windows 10 this works perfectly, on a docker image based on Ubuntu 20.04 I get the following error:

```
Warning: Error in .jcall: RcallMethod: cannot determine object class
  [No stack trace available]
```

This is my dockerfile, as near as I can tell I've got all dependencies sorted:

```

FROM rocker/r-ubuntu:20.04

# system and R libraries 
RUN apt-get update && apt-get install -y  --no-install-recommends \
gdal-bin \
git-core \
libcairo2-dev \
libcurl4-openssl-dev \
libgdal-dev \
libgeos-dev \
libgeos++-dev \
libgit2-dev \
libssh2-1-dev \
libssl-dev \
libudunits2-dev \
libxml2-dev \
make \
pandoc \
pandoc-citeproc \
unixodbc-dev \
zlib1g-dev \
default-jdk \
r-cran-rjava \
r-cran-shiny \
r-cran-glue \
r-cran-httr \
r-cran-jsonlite \
r-cran-data.table \
r-cran-lubridate \
r-cran-magrittr \
r-cran-shinydashboard \
r-cran-shinywidgets \
r-cran-shinyfiles \
r-cran-shinyjs \
r-cran-rsqlite \
r-cran-tidyverse \
r-cran-future \
r-cran-furrr \
r-cran-dt \
r-cran-ggfittext \
r-cran-formattable \
r-cran-odbc \
r-cran-pool \
r-cran-clipr \
r-cran-leaflet \
r-cran-styler \
r-cran-devtools \
r-cran-sf \
r-cran-geojsonsf \
r-cran-leafpop \
r-cran-lwgeom \
r-cran-satellite \
r-cran-leafsync \
r-cran-spatstat \
r-cran-spatialwidget \
&& rm -rf /var/lib/apt/lists/*

RUN R CMD javareconf

# R packages
RUN R -e "install.packages(c('treemapify','shinydashboardPlus','here','readwritesqlite','XLConnect'), repos = c('CRAN' = 'https://cran.rstudio.com/'), pkgtype='both', Ncpus=4, dependencies=TRUE)"

ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64

# copy the app to the image
RUN mkdir /root/my_app
RUN mkdir /root/my_app/data
COPY my_app /root/noc_triage
COPY /my_app/my_report.xlsx /root/my_app/data
COPY Rprofile.site /usr/lib/R/etc/

EXPOSE 3838

CMD ["R", "-e", "shiny::runApp('/root/my_app',host='0.0.0.0', port=3838)"]
````

**Comment by spoltier** [comment]
Hi @D3SL, 

Thank you for reporting this issue.

> My app loads in a local empty but preformatted workbook as a template, writes in data, and then outputs as a download a formatted report.

it would be useful to know at which step the error occurs:
* loading the workbook
* writing data
* saving the workbook (?)

Ideal would be a script that performs these steps, and can be used to reproduce the issue when running in an R session on the image built by that Dockerfile (minus the shiny app) - which I'm able to build.

**Comment by spoltier** [comment]
@D3SL we have released 1.0.4 which includes a previously missing java dependency. I'm closing this ticket as it has been open for over a year without feedback.
